core/src/main/kotlin/gui/Keeper.kt

```diff
@@ -0,0 +1,6 @@
+package gui
+
+object Keeper {
+    lateinit var game: gui.Game
```

Comment:
This object seems very strange. It shouln't be global, like all the functions you define. Many of the functions you define should beacome a methods of corresponding classes (i.e. Menu, Game or Keeper). It would be great if you will get rid of this object and all the free functions which modify the global state.

Reply:
removed